examples/v2/config_chat2.nim

```diff
@@ -0,0 +1,224 @@
+      desc: "Sets the log level."
+      defaultValue: LogLevel.INFO
+      name: "log-level" }: LogLevel
+    
+    nodekey* {.
+      desc: "P2P node private key as 64 char hex string.",
+      defaultValue: crypto.PrivateKey.random(Secp256k1, keys.newRng()[]).tryGet()
+      name: "nodekey" }: crypto.PrivateKey
+
+    listenAddress* {.
+      defaultValue: defaultListenAddress(config)
+      desc: "Listening address for the LibP2P traffic."
+      name: "listen-address"}: ValidIpAddress
+
+    tcpPort* {.
+      desc: "TCP listening port."
+      defaultValue: 60000
+      name: "tcp-port" }: Port
+
+    udpPort* {.
+      desc: "UDP listening port."
+      defaultValue: 60000
+      name: "udp-port" }: Port
+
+    portsShift* {.
+      desc: "Add a shift to all port numbers."
+      defaultValue: 0
+      name: "ports-shift" }: uint16
+
+    nat* {.
+      desc: "Specify method to use for determining public address. " &
+            "Must be one of: any, none, upnp, pmp, extip:<IP>."
+      defaultValue: "any" }: string
+    
+    ## Persistence config
+    
+    dbPath* {.
+      desc: "The database path for peristent storage",
+      defaultValue: ""
+      name: "db-path" }: string
+    
+    persistPeers* {.
+      desc: "Enable peer persistence: true|false",
+      defaultValue: false
+      name: "persist-peers" }: bool
+    
+    persistMessages* {.
+      desc: "Enable message persistence: true|false",
+      defaultValue: false
+      name: "persist-messages" }: bool
+
+    ## Relay config
+    
+    relay* {.
+      desc: "Enable relay protocol: true|false",
+      defaultValue: true
+      name: "relay" }: bool
+    
+    rlnRelay* {.
+      desc: "Enable spam protection through rln-relay: true|false",
+      defaultValue: false
+      name: "rln-relay" }: bool
+    
+    staticnodes* {.
+      desc: "Peer multiaddr to directly connect with. Argument may be repeated."
+      name: "staticnode" }: seq[string]
+    
+    keepAlive* {.
+      desc: "Enable keep-alive for idle connections: true|false",
+      defaultValue: false
+      name: "keep-alive" }: bool
+
+    topics* {.
+      desc: "Default topics to subscribe to (space separated list)."
+      defaultValue: "/waku/2/default-waku/proto"
+      name: "topics" .}: string
+
+    ## Store config
+
+    store* {.
+      desc: "Enable store protocol: true|false",
+      defaultValue: true
+      name: "store" }: bool
+
+    storenode* {.
+      desc: "Peer multiaddr to query for storage.",
+      defaultValue: ""
+      name: "storenode" }: string
+    
+    ## Filter config
+
+    filter* {.
+      desc: "Enable filter protocol: true|false",
+      defaultValue: false
+      name: "filter" }: bool
+    
+    filternode* {.
+      desc: "Peer multiaddr to request content filtering of messages.",
+      defaultValue: ""
+      name: "filternode" }: string
+    
+    ## Swap config
+
+    swap* {.
+      desc: "Enable swap protocol: true|false",
+      defaultValue: false
```

Comment:
I think the swap protocol should be enabled by default here. To support the changes that are currently made to the change log.

Reply:
Thanks! Done in f8505c8